students/jaehong/users/views.py

```diff
@@ -0,0 +1,43 @@
+import json, re
+
+from .models import User
+
+from django.http import JsonResponse
+from django.views import View
+
+
+class SignUpView (View):
+    def post(self, request):
+        try:
+            data= json.loads(request.body)
+            
+            name = data["name"]
+            email = data["email"]
+            password = data["password"]
+            phone_number = data["phone_number"]
+                
+            if User.objects.filter(email=email).exists() : # 기존 가입한 회원의 이메일과 가입할려는 사람의 이메일이 같을시 에러 반환
```

Comment:
이 정도 기능에서는 주석을 굳이 달아주지 않아도 될 것 같습니다!
```suggestion
            if User.objects.filter(email=email).exists():
```